.github/workflows/build-workflow.yml

```diff
@@ -42,3 +44,13 @@ jobs:
           TAG_NAME: upstream-3-tag-${{ steps.date.outputs.date }}
           RELEASE_NAME: upstream-3-name-${{ steps.date.outputs.date }}
 
+  upstream-trigger:
+    runs-on: ubuntu-latest
+    needs: [ upstream-3 ]
+    uses: pshynin/gha-test/.github/workflows/test-reusable-workflow.yml@v1
```

Comment:
```suggestion
    uses: pshynin/gha-test/.github/workflows/test-reusable-workflow.yml@test4
```
maybe?

Reply:
well.. this should be some static value i guess, otherwise how it will be triggered on a new branch 🙄

Reply:
maybe this workflow has to be on master first.. then it will work 🤔🤔🤔

Reply:
yeah that makes sense, just saying that `v1` tag doesn't exist https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_iduses